AlphaWallet/Protection/Coordinators/SplashCoordinator.swift

```diff
@@ -9,13 +9,16 @@ class SplashCoordinator: Coordinator {
 
     init(window: UIWindow) {
         self.window = window
+        window.backgroundColor =  .white;
     }
 
     func start() {
         window.rootViewController = SplashViewController()
         window.isHidden = false
+        window.rootViewController = SplashViewController()
```

Comment:
Why is it being created and assigned twice?

Reply:
it was by mistake removing it. one part of it was commented.